routers/private/add_track.py

```
@@ -0,0 +1,150 @@
+    playlist_name = context["playlist_name"]
+
+    audio_file_id = get_audio_file_id(message)
+    audio_title = get_audio_title(message)
+
+    track_added = add_track(playlist_name,user_db_id,audio_file_id)
+    if track_added is None:
+        logger.error(f"Failed to add '{audio_title}' to {playlist_name} for user '{user_id}'.",exc_info=True)
+        await message.answer(
+            f"❌ Failed to add '{audio_title}' to {playlist_name}."
+        )
+    elif track_added == False:
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not track_added:` for false checks

Replace with `not track_added`

(E712)